python/gaas_server/gaas_handler.py

```diff
@@ -99,8 +164,16 @@ def call_graph_creation_extension(self, func_name,
                 if func is not None:
                     func_args = eval(func_args_repr)
                     func_kwargs = eval(func_kwargs_repr)
+
+                    # All graph creation extensions are passed a
+                    # ExtensionServerFacade instance as the first arg to allow
+                    # them to query the "server" in a safe way, if needed.
+                    gaas_server_facade = ExtensionServerFacade(self)
+
                     try:
-                        graph_obj = func(*func_args, **func_kwargs)
+                        graph_obj = func(gaas_server_facade,
+                                         *func_args,
+                                         **func_kwargs)
```

Comment:
This change assumes `*_extension.py` signatue has an argument for  `gaas_server_facade` which is unexpected to me . 

For example our old reddit extenstion (see beow) will fail here as it does not expect  `gaas_server_facade`. 

```python
def create_property_graph_from_reddit_data(dataset_path):
    """
    Creates and returns a cugraph PropertyGraph using data from
    reddit_graph_file_name and reddit_data_file_name.
    """
    print(dataset_path)
    dataset_path_obj = str(Path(dataset_path).absolute())
    reddit_graph_file_name = dataset_path_obj+"/reddit_graph.npz"
    reddit_data_file_name = dataset_path_obj+"/reddit_data.npz"

        
    coo_adj = sp.load_npz(reddit_graph_file_name)
    edgelist = cudf.DataFrame()
    edgelist['src'] = cudf.Series(coo_adj.row)
    edgelist['dst'] = cudf.Series(coo_adj.col)
    edgelist['wt'] = cudf.Series(coo_adj.data)

    reddit_data = np.load(reddit_data_file_name)
    features = reddit_data["feature"]
    cu_features = cudf.DataFrame(features)
    cu_features['name'] = np.arange(cu_features.shape[0])

    pG = PropertyGraph()
    pG.add_edge_data(edgelist, vertex_col_names=("src", "dst"))
    pG.add_vertex_data(cu_features, vertex_col_name="name")

    return pG
```

Reply:
Something changed in one of the branches I merged with that made gaas_server_facade a required argument.  @rlratzel do you know why this is the case now?

Reply:
it's ok, I already updated to work with the new version.